cumulusci/utils/yaml/tests/test_cumulusci_yml.py

```diff
@@ -37,3 +43,59 @@ def test_converter_is_selective(self):
 
         rc = _replace_nbsp(inp)
         assert rc == inp
+
+    def test_invalid_cumulusci_yml_file(self):
+        with temporary_dir() as temp_dir:
+            cumulusci_yml_filepath = Path(temp_dir) / "cumulusci.yml"
+
+            with open(cumulusci_yml_filepath, "w+") as cumulusci_yml:
+                invalid_yml = """xyz: abc   \n>>>\nefg: lmn\n"""
+                cumulusci_yml.write(invalid_yml)
+                cumulusci_yml.seek(0)
+
+                with pytest.raises(CumulusCIException) as error:
+                    cci_safe_load(cumulusci_yml)
+
+        assert error.typename == "CumulusCIException"
```

Comment:
This is unnecessary -- the `pytest.raises` context manager already asserts that the expected exception was raised. You can also pass `matches` to `pytest.raises` to assert that a string is included in the except message, instead of a separate assertion, if you want.